core/pkg/opaprocessor/utils.go

```diff
@@ -106,6 +106,6 @@ var imageNameNormalizeDefinition = func(bctx rego.BuiltinContext, a *ast.Term) (
 	if err != nil {
 		return nil, fmt.Errorf("invalid parameter type: %v", err)
 	}
-	normalizedName, err := normalize_image_name(string(aStr))
+	normalizedName, err := cautils.Normalize_image_name(string(aStr))
```

Comment:
@YiscahLevySilas1 can this affect the controls that are using this function?

Reply:
I don't think it will affect them